TFT/src/User/Configuration.h

```diff
@@ -174,7 +178,7 @@
  * screen to prevent printer idling and stuttering  due to empty printer buffer.
  * Note: this might make the  graphics slow when switching menus while printing.
 */
-//#define RAPID_SERIAL_COMM
+#define RAPID_SERIAL_COMM

```

Comment:
My experience is that the microcontroller in the TFT is more than fast enough to enable this without any observed UI slowness and it really ought to be enabled. Otherwise you WILL get stuttering during printing. Even with this enabled, I'm noticing some stuttering when printing in touch mode. I think there's still a lot of room for optimization.